.github/workflows/check-patch.yml

```diff
@@ -41,9 +40,18 @@ jobs:
           find tmp.repos -iname \*rpm -exec mv "{}" exported-artifacts/ \;
           mv ./*tar.gz exported-artifacts/
 
-    - name: test install
+    - name: install master built rpm
+      if: contains(github.ref, "ref/head/master")
       run: |
           yum install -y exported-artifacts/ovirt-release-master-4*noarch.rpm
+
+    - name: install 4.4 built rpm
+      if: contains(github.ref, "ref/head/ovirt-4\.4")
+      run: |
+          yum install -y exported-artifacts/ovirt-release44-4*noarch.rpm
```

Comment:
Can we just install `exported-artifacts/*.rpm`? If that's not possible, please add a comment explaining the reasoning behind this logic.

Reply:
Added comment